src/Fixer/Basic/Psr0Fixer.php

```diff
@@ -92,7 +99,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             $path = str_replace('\\', '/', $file->getRealPath());
             $dir = dirname($path);
 
-            if (isset($this->configuration['dir'])) {
+            if ($this->configuration['dir']) {
```

Comment:
Yes I meant `'0'`.

Indeed, if we consider `'0'` an invalid configuration value, this should be checked while resolving configuration. But why would we consider it invalid?